rudder-lang/Makefile

```diff
@@ -1,5 +1,19 @@
-build:
-	cargo build
+.DEFAULT_GOAL := build
+SHELL = /bin/bash
+
+cargo:
+	curl https://sh.rustup.rs -sSf | sh -s -- -y
+	source $(HOME)/.cargo/env
```

Comment:
useless since each line is independant

Reply:
yes, a remainder i forgot to remove, will do. This Makefile is in a temporary state for now anyway, it's a base for the upcoming integration work